pacman-nodejs-app/tools/migrate/awzkmt.sh

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env bash
```

Comment:
No I didn't. I only worked through the manual migration/documentation so far. I am working on the kmt tool today.